src/components/ProfilePage/AddressBox/AddressBox.tsx

```diff
@@ -52,12 +52,12 @@ export const AddressBox = ({ address, isEVM, imageLink }: AddressBoxProps) => {
     ? imageLink
     : address && isEVM
       ? `https://effigy.im/a/${address}.png`
```

Comment:
@admsk2 can you implement a logic so when this url is not reponsive we show the default one pls?